doc/nrf/app_dev/device_guides/nrf70/wifi_advanced_security_modes.rst

```diff
@@ -0,0 +1,78 @@
+.. _ug_nrf70_wifi_advanced_security_modes:
+
+nRF70 Series advanced security modes
+####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Enterprise security
+*******************
+
+The nRF70 Series devices support Wi-Fi enterprise security, which is a more secure form of Wi-Fi security compared to Wi-Fi personal security.
+Wi-Fi enterprise security is used in corporate environments where the security requirements are more stringent.
+It is based on the IEEE 802.1X standard, which defines the port-based network access control.
+
+The nRF70 Series devices support the following Wi-Fi enterprise security mode, ``WPA2-EAP-TLS``.
+This mode uses the Extensible Authentication Protocol (EAP) with Transport Layer Security (TLS) for authentication.
+The client and the authentication server exchange certificates to authenticate each other.
+
+
+Enterprise testing: X.509 certificate headers generation
+========================================================
+
+Wi-Fi enterprise security requires use of X.509 certificates.
+Test certificates in PEM format are available at :zephyr_file:`samples/net/wifi/test_certs/` repository.
+During the build process, the certificates are converted to a C header file that is included in the Wi-Fi shell
+module or the :ref:`Wi-Fi credentials <lib_wifi_credentials>` library.
```

Comment:
```suggestion
During the build process, the certificates are converted to a C header file that is included in the Wi-Fi shell module or the :ref:`Wi-Fi credentials <lib_wifi_credentials>` library.
```

Reply:
Thanks, I will take this up in next push or if there is no push (here's hoping) in a subsequent PR.